widgetastic_manageiq/__init__.py

```diff
@@ -4881,6 +4887,30 @@ def is_plan_succeeded(self, plan_name):
         except NoSuchElementException:
             return False
 
+    def edit_plan(self, plan_name, cancel=False):
+        """Edit the plan by its name"""
+        try:
+            el = self._get_plan_element(plan_name)
+            self.browser.click(self.ITEM_KEBAB_DROPDOWN_LOCATOR, parent=el)
+            self.root_browser.click(self.ITEM_EDIT_BUTTON_LOCATOR)
+            self.root_browser.click(self.INFRA_MAP_LOCATOR)
```

Comment:
Nadeem , edit_plan method in widgetastic should just click on `Edit Plan` in kebab dropdown . 
Rest of editing,  like selecting new infra mapping and other fields should be down in migration_plan.py as it already has those fields defined .
Please write a method 'edit_plan' in migration_plans.py which will call `edit_plan` in widgetastic file and select the kebab menu and then like any update method rest of the work wil be done in migration_plan.py

Reply:
Example :      
 
     def edit_plan(self, plan_name, cancel=False):
     """Edit the plan by its name"""
        try:
            el = self._get_plan_element(plan_name)
            self.browser.click(self.ITEM_KEBAB_DROPDOWN_LOCATOR, parent=el)
            self.root_browser.click(self.ITEM_EDIT_BUTTON_LOCATOR)
            return True
      except NoSuchElementException:
            return False